src/main/java/klerer/projectile/ProjectileGraph.java

```diff
@@ -0,0 +1,48 @@
+package klerer.projectile;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class ProjectileGraph extends JComponent {
+
+    private Projectile projectile = new Projectile(0, 0);
+
+    @Override
+    protected void paintComponent(Graphics g) {
+        super.paintComponent(g);
+
+        g.translate(0, getHeight());
+
+        g.setColor(Color.BLACK);
+
+        projectile.setSeconds(0);
+
+        for (int i = 0; i <= (projectile.getApexTime() * 2) + 1; i += 1) {
+            double currentX = projectile.getX();
+            double currentY = projectile.getY();
+
+            projectile.setSeconds(i);
+
+            g.drawLine((int) currentX,
+                    -(int) currentY,
+                    (int) projectile.getX(),
+                    -(int) projectile.getY());
+        }
+
+        g.setColor(Color.BLUE);
+        projectile.getPeakY();
+        g.fillOval((int) (projectile.getInterceptX() / 2 - 3.5),
```

Comment:
Why -3.5?

Reply:
so that the circle will be in the middle of the line and not on the bottom
without it looks like this:
<img width="414" alt="Screenshot 2024-02-28 at 2 10 13 PM" src="https://github.com/hklerer/klerer-projectile-2024/assets/126403766/a33de982-f69d-42de-acf8-e2c2db7664b2">


with it looks like this:
<img width="446" alt="Screenshot 2024-02-28 at 2 10 33 PM" src="https://github.com/hklerer/klerer-projectile-2024/assets/126403766/644d6b9e-d250-45ec-add9-a7d763c95a75">

Reply:
Where did you get 3.5 from?

Reply:
I tried different numbers and it was what made it in the middle

Reply:
It should be half the width/height of the circle.